tests/test_grammar.py

```diff
@@ -227,10 +253,14 @@
     # Interpolations in quoted strings.
     ("str_quoted_inter", "'${null}'", "None"),
     ("str_quoted_esc_single_1", r"'ab\'cd\'\'${str}'", "ab'cd''hi"),
-    ("str_quoted_esc_single_2", "'\"\\\\\\\\\\${foo}\\ '", r'"\${foo}\ '),
+    ("str_quoted_esc_single_2", r"""'"\\\\\${foo}\ '""", r'"\${foo}\ '),
```

Comment:
Can you explain this one? I find it surprising that we need 5 slashes before the $ and 1 slash after the }.
Is this something that will be addressed by the followup double escaping diff? if so, what will it look like after we eliminate the double unescaping?